shuffle/cli/README.md

```diff
@@ -5,6 +5,13 @@ blockchain.
 
 ## Setup
 
+Prerequisites:
+
+```
+Install brew: https://brew.sh/
```

Comment:
I think this is not required.
User just need to follow installation doc for deno